lalrpop/src/api/mod.rs

```diff
@@ -158,6 +158,12 @@ impl Configuration {
         self
     }
 
+    /// Set the max macro recursion depth.  As lalrpop is resolving a macro, it may discover new macros uses in the macro definition to resolve.  Typically deep recursion indicates a recursive macro use that is non-resolvable.  The default resolution depth is 200.
```

Comment:
It's really a small nitpick, but do we want to hardcode the default value in the comment? That's one more things to make sure to keep in sync, while I feel like it doesn't add much to the understanding of this recursion limit. One can always go to the definition of the `macro_recursion_limit` or hopefully grep their way to find the current default value set.

```suggestion
    /// Set the max macro recursion depth.  As lalrpop is resolving a macro, it may discover new macros uses in the macro definition to resolve.  Typically deep recursion indicates a recursive macro use that is non-resolvable.
```

Reply:
Listing the default is consistent with all the other config functions that take an argument.  I'm definitely sympathetic to the "single point of truth" case to omit it, but documentation is much more visible to users than grepping through source files, and I think including the defaults in the docs add value.

Reply:
Fair enough. Works for me, then